validator/templates/validator/validate_metadata_client.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'catalog/base.html' %}
+{% load static %}
+
+{% block title %}Metadata Template Validation{% endblock %}
+
+{% block desc %}
+  <script type="application/javascript">
+      const on_gae = {{ is_pgs_app_on_gae }};
+  </script>
+  {% if is_pgs_app_on_gae %}
+    <script defer type="module" src="{% static 'validator/js/metadata_consumer.min.js' %}"></script>
+  {% else %}
+    <script defer type="module" src="{% static 'validator/js/metadata_consumer.js' %}"></script>
+  {% endif %}
+{% endblock %}
+
+{% block content %}
+    <nav aria-label="You are here:">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="/curation_tracker/static">PGS Catalog</a></li>
+        <li class="breadcrumb-item">Metadata Template Validation</li>
+      </ol>
+    </nav>
+    
+    <style>
```

Comment:
Is this something we need to move to the main `pgs.scss` file as it is duplicated in the scoring file validator template ?

Reply:
It is unused in the scoring files template, I will remove it from there

Reply:
I will be better in pgs.scss anyway as it uses the variables for the colour